idact/detail/add_cluster_app/main.py

```diff
@@ -77,13 +74,27 @@ def main(cluster_name: str,
     load_environment()
 
     log.info("Adding cluster...")
-    cluster = add_cluster(name=cluster_name,
-                          user=user,
-                          host=host,
-                          port=port,
-                          auth=auth,
-                          key=key,
-                          install_key=install_key)
+
+    if auth == 'PUBLIC_KEY':
+        auth_method = AuthMethod.PUBLIC_KEY
+        key_type = KeyType.RSA
+        cluster = add_cluster(name=cluster_name,
+                              user=user,
+                              host=host,
+                              port=port,
+                              auth=auth_method,
+                              key=key_type,
+                              install_key=install_key)
+    elif auth == 'ASK_EVERYIME':
+        auth_method = AuthMethod.ASK
+        cluster = add_cluster(name=cluster_name,
```

Comment:
Maybe there is an option to place `add_cluster` method in code only once - after if-elif and key for ASK_EVERYTIME would be None?

Reply:
nice idea